resources/lang/en/wiki.php

```diff
@@ -12,6 +12,7 @@
         'missing_translation' => 'Requested page could not be found for currently selected language.',
         'needs_cleanup_or_rewrite' => 'This page does not meet the standards of the osu! wiki and needs to be cleaned up or rewritten. If you are able to help out, please consider updating the article!',
         'search' => 'Search existing pages for :link.',
+        'stub' => 'This article is a stub. You can help osu! wiki by expanding it.',
```

Comment:
Reads pretty weirdly.

- If it's going to directly say "you can help" then it needs to be a link
- The term "stub" doesn't mean much to a lot of people I think. "Incomplete" feels better to me.

Maybe something like "This article is incomplete and waiting on someone to expand it"?

Reply:
making it a link was my original idea, as well as adding it to the `outdated`/`needs_cleanup` articles. I couldn't decide what to reference, the article on GitHub, or `CONTRIBUTING. md`, or https://osu.ppy.sh/wiki/OWCG (the latter would be more fair, but it's not in the best shape right now)